DuckDuckGo/PassKitPreviewHelper.swift

```diff
@@ -42,3 +44,45 @@ class PassKitPreviewHelper: FilePreview {
         }
     }
 }
+
+class ZippedPassKitPreviewHelper: FilePreview {
```

Comment:
Why not create this in a new file?

Reply:
My thinking is:
* new file isn't really that useful (especially if you use Xcode to navigate the symbols)
* two highly related things are right next to each other

Don't feel too strongly though